### PR TITLE
Improve status messages

### PR DESCRIPTION
**Pull request by siman**
Use new beautiful components for status messages instead of `<em>...</em>`.

### CODE REVIEW

**Comment by mnaamani** [comment]
This is where the bug which is fixed in https://github.com/Joystream/apps/pull/318/commits/75c7c65dce2cf64a984d829e3e0b0122d7044ba4 was introduced.
would adding `withApi` at the top level better to avoid missing it in other components when they get added.

Here: 

https://github.com/Joystream/apps/blob/17ac93c2d58b7ef39a84e0502201f7d63aa87caf/packages/joy-media/src/index.tsx#L85

File: packages/joy-media/src/video/PlayVideo.view.tsx [hunk 1]
```diff
@@ -18,10 +19,11 @@ export const PlayVideoView = MediaView<Props>({
 
     const channelId = new ChannelId(video.channelId)
     const channel = await transport.channelById(channelId)
+    const moreChannelVideos = (await transport.videosByChannelId(channelId, 5)).filter(x => x.id !== video.id)
```
Comment (Lezek123) [inline packages/joy-media/src/video/PlayVideo.view.tsx:22]: This is probably not the best approach, as it causes the number of displayed videos to vary depending on whether we are viewing one of the first 5 `videosByChannelId` or not.

Comment (mnaamani) [reply packages/joy-media/src/video/PlayVideo.view.tsx:22]: Good observation. Feel free to improve on it.

File: packages/joy-media/src/Upload.tsx [hunk 2]
```diff
@@ -313,6 +314,6 @@ class Component extends React.PureComponent<Props, State> {
 export const UploadWithRouter = withMulti(
   Component,
   translate,
-  withOnlyMembers,
+  withMembershipRequired,
```
[the remark this answers is not in the document]
Comment (mnaamani) [reply packages/joy-media/src/Upload.tsx:317]: So what is the best way to fix this to make sure api is always available where needed and reduces possibility of forgetting to add `withApi` ?

[the remark this answers is not in the document]
Comment (Lezek123) [reply packages/joy-media/src/Upload.tsx:317]: `withApi` is quite rarely used in general, this was just a single bug, so I don't think any "global" changes are necessary. There are better ways of handling api interactions (through context for example). The problem here was that `withOnlyMembers` was providing both `ApiProps` and `MyAccountProps`, so after changing it, those should be removed from the `Props` type, which would reveal the error instantly.

**Pull request merged**